### PR TITLE
Add delete notes command

### PR DESCRIPTION
**Pull request by shuckycheese**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteNotesCommand.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+import seedu.address.model.Model;
+import seedu.address.model.person.Notes;
+import seedu.address.model.person.Person;
+
+/**
+ * Deletes Notes from the person identified using it's displayed index from the address book.
+ */
+public class DeleteNotesCommand extends Command {
+
+    public static final String COMMAND_WORD = "delete_notes";
```
Comment (kyouscrap) [inline src/main/java/seedu/address/logic/commands/DeleteNotesCommand.java:20]: might want to adhere to the new command format of no _

File: src/main/java/seedu/address/logic/commands/DeleteNotesCommand.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes the patient notes from the person identified by the index number used in the "
+            + "displayed person list.\n"
+            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1";
```
Comment (kyouscrap) [inline src/main/java/seedu/address/logic/commands/DeleteNotesCommand.java:26]: might want to specify that you should only delete notes from someone who has existing notes

File: src/main/java/seedu/address/logic/commands/DeleteNotesCommand.java (hunk 1 above)
Comment (choonzies) [reply src/main/java/seedu/address/logic/commands/DeleteNotesCommand.java:20]: Good catch!

File: src/main/java/seedu/address/logic/commands/DeleteNotesCommand.java (hunk 2 above)
Comment (choonzies) [reply src/main/java/seedu/address/logic/commands/DeleteNotesCommand.java:26]: We will need to make our User Guide elaborate on these nitty gritty stuff too in the future

**Review by choonzies — CHANGES_REQUESTED** [review]
Minor changes as requested by @kyouscrap 

Implementation and test cases LGTM.

File: src/main/java/seedu/address/logic/commands/DeleteNotesCommand.java (hunk 1 above)
Comment (shuckycheese) [reply src/main/java/seedu/address/logic/commands/DeleteNotesCommand.java:20]: That's a good point but i was planning to merge it first than change it with the other command formats.

Comment (shuckycheese) [reply src/main/java/seedu/address/logic/commands/DeleteNotesCommand.java:20]: I think ill change it here and just make this PR close both issues

**Pull request merged**